src/sorcha/modules/PPMiniDifi.py

```diff
@@ -78,25 +81,34 @@ def hasTracklet(mjd, ra, dec, maxdt_minutes, minlen_arcsec):
     ## tracklets by taking all observations in a night and computing
     ## all of theirs pairwise distances, then selecting on that.
     nobs = len(ra)
+    if nobs >= 1 and min_observations == 1:  # edge case when 1 observation is needed for a tracklet
+        return True
     if nobs < 2:
```

Comment:
This doesn't cover if one needs three observations for a tracklet 
I think it's 
if nobs < 2 needs to be nobs  min_observations